test/controllers/works_controller_test.rb

```diff
@@ -0,0 +1,46 @@
+require "test_helper"
+
+describe WorksController do
+  let(:work) { works(:one) }
```

Comment:
These tests produced a number of errors. I think to access a variable when using `let`, you don't use the `@` symbol.  

```
Minitest::UnexpectedError:         ActionController::UrlGenerationError: No route matches {:action=>"edit", :controller=>"works", :id=>nil}, missing required keys: [:id]
            test/controllers/works_controller_test.rb:30:in `block (2 levels) in <main>'
```